components/shared/right-sidebar/RightSidebar.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+'use client';
+import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import photo01 from '@/public/photo01.svg';
+import SidebarTags from './SidebarTags';
+import SidebarRelatedPosts from './SidebarRelatedPosts';
+import SidebarOnThisPage from './SidebarOnThisPage';
+import SidebarSocialMedia from './SidebarSocialMedia';
+
+
+
+const RightSidebar = () => {
+
+  return (
+    <div className='custom-scrollbar bg-black-800 flex-2 sticky right-0 top-0 z-20 flex h-screen flex-col overflow-y-auto p-6 pt-36 lg:left-0 lg:w-60'>
+      <Link href="/profile" className='flex flex-row space-x-2 pb-12'>
+        <Image
+          src={photo01}
+          alt="Profile photo"
+        />
+        <div className='flex flex-col'>
+          <p className='paragraph-3-medium'>
+            Nikky Eva
+          </p>
+          <p className='text-white-300 paragraph-4-regular'>
+            nikky@jsmastery.pro
+          </p>
+        </div>
+      </Link>
+
+
+      <SidebarTags/>
+      {/* <SidebarRelatedPosts/> */}
+      {/* <SidebarOnThisPage/> */}
+      {/* <SidebarSocialMedia/> */}
```

Comment:
A different version of the sidebar seems to be rendered in different areas of the app - created a component for each to keep track of differences